# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../dietlibc/pkg_patch/pkg_binutils.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# Copyright (C) 1998 - 2004 ROCK Linux Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- ./bfd/elf.c.orig	Tue Apr 30 19:19:55 2002
+++ ./bfd/elf.c	Tue Apr 30 19:21:08 2002
@@ -5872,8 +5872,10 @@
    machine-dependent procfs structure in order to pick
    out details about the corefile.  */
 
+#ifndef __dietlibc__
 #ifdef HAVE_SYS_PROCFS_H
 # include <sys/procfs.h>
+#endif
 #endif
 
 /* FIXME: this is kinda wrong, but it's what gdb wants.  */
--- ./libiberty/regex.c~	Fri Sep 27 17:33:09 2002
+++ ./libiberty/regex.c	Sun Dec 29 04:47:22 2002
@@ -302,7 +302,7 @@
 
 /* Integer type for pointers.  */
 # if !defined _LIBC && !defined HAVE_UINTPTR_T
-typedef unsigned long int uintptr_t;
+/*typedef unsigned long int uintptr_t;*/
 # endif
 
 /* Should we use malloc or alloca?  If REGEX_MALLOC is not defined, we
